fastlane/codecov.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+bash <(curl -s https://codecov.io/bash)
+sleep 5
```

Comment:
Might be worth a comment above the sleep, though I suspect most people will / can guess what it might be for :)